.github/workflows/deploy-to-netlify.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 'lts/*'
+          node-version: '18.14.2'
```

Comment:
If you use `node-version-file`, you can reduce the description of the version to one time.

https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

Reply:
I didn't know that. I will fix it. Thank you!

Reply:
fixed: https://github.com/line/liff-playground/pull/19/commits/8fbb351ae5e4e5c355f04491436ea5b7a8f36a63